File: src/MyHealth.Client.Cordova/content/app/modules/homeAppointments/models/Patient.ts
module MyHealth.Client.Cordova.Application.HomeAppointments {
    export class Patient {
        address: string;
        age: number;
        bloodType: string;
        clinicId: string;
        dateOfBirth: Date;
        email: string;
        gender: string;
        height: number;
        id: string;
        name: string;
        patientId: number;
        phone: string;
        picture: string;
        tenantId: number;
        weight: number;

        deserialize(data : any) {
            this.address = data.address;
            this.age = data.age;
            this.bloodType = data.bloogType;
            this.clinicId = data.clinicId;
            this.dateOfBirth = new Date(data.dateOfBirth) || data.dateOfBirth;
            this.email = data.email;
            this.gender = data.gender;
            this.height = data.height;
            this.id = data.id;
            this.name = data.name;
            this.patientId = data.patientId;
            this.phone = data.phone;
            this.picture = data.picture;
            this.tenantId = data.tenantId;
            this.weight = data.weight;
            return this;
        }
    }
}

File: src/MyHealth.Client.Core/Model/Patient.cs
using System;
using System.Collections.Generic;

namespace MyHealth.Client.Core.Model
{
    public class Patient
    {
        public int PatientId { get; set; }

        public string Name { get; set; }

        public string ClinicId { get; set; }

        public Gender Gender { get; set; }

        public double Weight { get; set; }

        public double Height { get; set; }

        public string BloodType { get; set; }

        public string Address { get; set; }

        public string Email { get; set; }

        public int Age { get; set; }

        public DateTime DateOfBirth { get; set; }

        public string Phone { get; set; }

        public byte[] Picture { get; set; }

        public bool Deleted { get; set; }

        public int InsuranceCompanyId { get; set; }

        public int TenantId { get; set; }

        public ICollection<Doctor> Doctors { get; set; }

        public ICollection<ClinicAppointment> ClinicAppointments { get; set; }

        public ICollection<Medicine> Medicines { get; set; }

        public ICollection<HomeAppointment> HomeAppointments { get; set; }

    }
}
